src/test/java/io/confluent/connect/jdbc/JdbcSourceTaskUpdateTest.java

```diff
@@ -191,7 +214,7 @@ public void testTimestampWithDelay() throws Exception {
 
     db.insert(SINGLE_TABLE_NAME, "modified", JdbcUtils.formatUTC(new Timestamp(10L)), "id", 1);
 
-    startTask("modified", null, null, 4L);
+    startTask("modified", null, null, null, 4l);
```

Comment:
Nit: the `L` is preferred since it's not easily mistakable for `1` like `l` can be.